application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -899,6 +908,14 @@ type LegTime {
   scheduledTime: OffsetDateTime!
 }
 
+"A level with a name and comparable number. Levels can sometimes contain half levels, e.g. '1.5'."
+type Level {
+  "0-based comparable number."
+  level: Float!
+  "Name if it exists, e.g. 'M' or 'P1'."
+  name: String!
```

Comment:
I wonder if we should call this name or code or publicCode. This is similar to entrances/stops etc. Does OSM reference limit these to be short names like the examples or can it be some "Amazing shopping mall's level 3"?